src/fileimpl.h

```diff
@@ -89,6 +90,7 @@ namespace zim
       const Fileheader& getFileheader() const  { return header; }
       zsize_t getFilesize() const;
       bool hasNewNamespaceScheme() const { return m_newNamespaceScheme; }
+      bool hasFullTitleIndex() const { return m_hasFullTitleIndex; }
```

Comment:
Doesn't it make sense to signal the presence of a front article index instead? As far as I understand the full title index should be always present in a ZIM archive, but it is "overriden" by the front article index if the latter is available.

Reply:
Yes, as it is a internal method I haven't think a lot on the api (better to discuss in #514).
But your remark is right. And is is more coherent with `hasNewNamespaceScheme` which return `true` if we have something "new".